pkg/utils/utils.go

```diff
@@ -70,3 +71,17 @@ func DecompressGZIP(compressedData []byte) (decompressedData []byte, err error)
 
 	return decompressedData, nil
 }
+
+func HumanReadableSize(data []byte) string {
```

Comment:
You probably should add a unit test for this routine just to increase our unit test coverage and validate that it is working as expected.